pq-crypto/sike_r3/sikep434r3.h

```diff
@@ -33,15 +33,15 @@ uint64_t bswap64(uint64_t x);
 /* Arch specific definitions */
 #define digit_t S2N_SIKE_P434_R3_NAMESPACE(digit_t)
 #define hdigit_t S2N_SIKE_P434_R3_NAMESPACE(hdigit_t)
-#if defined(_AMD64_) || defined(__x86_64) || defined(__aarch64__) || defined(_S390X_) || defined(_ARM64_) || defined(__powerpc64__) || (defined(__riscv) && (__riscv_xlen == 64))
+#if defined(_AMD64_) || defined(__x86_64) || defined(__x86_64__) || defined(__aarch64__) || defined(_S390X_) || defined(_ARM64_) || defined(__powerpc64__) || (defined(__riscv) && (__riscv_xlen == 64))
```

Comment:
This is getting pretty intense. Is there any other way to detect 32-bit vs 64-bit? Could you use cmake + CMAKE_SIZEOF_VOID_P instead? Or maybe do something with s2n_init + sizeof?